src/pages/components/style.scss

```diff
@@ -31,10 +31,11 @@
     color: $other-light;
 }
 
-.text-input-box h3,
-p {
```

Comment:
Here you are change margins for every "p" and "h3", try select theses elements by classes to avoid interference in something else.

Reply:
![image](https://user-images.githubusercontent.com/62728106/162769838-1158551d-eb0c-4471-9510-6917db72722a.png)
Done